spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/logging/StackdriverLoggingAutoConfiguration.java

```diff
@@ -42,6 +44,7 @@
 @ConditionalOnClass({HandlerInterceptorAdapter.class, LoggingAppender.class})
 @ConditionalOnMissingBean(type = "org.springframework.cloud.sleuth.autoconfig.SleuthProperties")
 @AutoConfigureAfter(StackdriverTraceAutoConfiguration.class)
+@ConditionalOnWebApplication(type = Type.SERVLET)
```

Comment:
Any chances to add a test-case with the non-`SERVLET` environment to be sure that this configuration is not applied and there is no any collisions: https://docs.spring.io/spring-boot/docs/2.0.3.RELEASE/reference/htmlsingle/#boot-features-testing-spring-boot-applications-detecting-web-app-type ?